recipes/regiis.rb

```diff
@@ -28,4 +28,5 @@
   command "%WINDIR%\\Microsoft.Net\\Framework64\\v4.0.30319\\aspnet_regiis.exe -i -enable > #{guard_file}"
   creates guard_file
   only_if 'sc.exe query W3SVC'
+  not_if { File.exist?(guard_file) }
```

Comment:
This doesn't actually replicate the same behavior. The not_if stops the entire execute resource from running where the creates doesn't stop the command or resource from running but only stops it from overwriting the file in place if it already exists. The command still runs with every chef-client run. 

The link you gave above doesn't say anything about preventing command from running just suppressing it's output to the filesystem when it does. Here is a quote from the link:

creates
Ruby Type: String

Prevent a command from creating a file when that file already exists.